v6/jbuild

```diff
@@ -32,10 +32,25 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
  ((name xcp_v6)
   (public_name xcp.v6)
   (flags (:standard -w -39 %s))
+  (modules (:standard \ v6_cli ))
   (libraries
    (rpclib
     threads
     xcp))
   (wrapped false)
   %s))
+
+(executable
+ ((name v6_cli)
+  (modules (v6_cli))
+  (libraries
+   (cmdliner
+    rpclib.cmdliner
+    rpclib.markdown
+    xcp.v6))))
+
```

Comment:
Not sure if there's a more succinct way to include rules for v6_cli, this is just based on the squeezed jbuild and how it builds `memory_cli.exe`

Reply:
You can create aliases, but it's not worth. Let's get rid of those empty runtests and add a section in the readme about the debugging cli tools that you can build

Reply:
The empty runtests were there to check that the examples/clis can be built even if not installed, my worry was is that if we don't build them they'll bitrot and when we do want to use them they won't work anymore. There's probably a nicer way to do that than using runtest though.

Reply:
I imagined something like that, but I'd rather add a post-install-hook to run `jbuilder build` for them, eventually adding an alias for it, like `@clitools` or something like that